software/controller/lib/core/sensors.cpp

```diff
@@ -128,11 +154,16 @@ Pressure Sensors::ReadPressureSensor(Sensor s) {
 }
 
 SensorReadings Sensors::GetReadings() {
```

Comment:
This function should probably be of the signature:
`SensorReadings Sensors::GetReadings() const`